ferrocene/doc/user-manual/src/rustfmt/usage.rst

```diff
@@ -0,0 +1,30 @@
+.. SPDX-License-Identifier: MIT OR Apache-2.0
+   SPDX-FileCopyrightText: The Ferrocene Developers
+
+Using ``rustfmt``
+=================
```

Comment:
This is missing `rustfmt --check`, and using it through `cargo fmt`.

Reply:
Should we mention `cargo fmt` even if it is not going to be qualified (yet)?

Reply:
Yes, we can add a note that `cargo fmt` is not qualified yet (as we do in the CLI page for rustc).

Reply:
`cargo fmt` has a few additional cli options (package, manifest-path etc.). Should we also document them?